### PR TITLE
Updates deployment configs to include the prefix `tekton-hub-`

### PR DESCRIPTION
**Pull request by pratap0007**
Deployed-link: https://tekton-hub-ui-hub-prefix.apps.cluster-z7wdc.z7wdc.sandbox286.opentlc.com/

Sigend-off-by: Shiv Verma <shverma@redhat.com

# Submitter Checklist

These are the criteria that every PR should meet, please check them off as you
review them:

- [ ] Includes [tests](https://github.com/tektoncd/community/blob/main/standards.md#principles) (if functionality changed/added)
- [ ] Run API Unit Tests, Lint Checks, API Design, Golden Files with `make api-check`
- [ ] Run UI Unit Tests, Lint Checks with `make ui-check`
- [ ] Commit messages follow [commit message best practices](https://github.com/tektoncd/community/blob/main/standards.md#commit-messages)

_See [the contribution guide](https://github.com/tektoncd/pipeline/blob/main/CONTRIBUTING.md) for more details._

### CODE REVIEW

**Review by vdemeester — APPROVED** [review]
/cc @piyush-garg

File: config/04-kubernetes/40-auth-ingress.yaml [hunk 1]
```diff
@@ -6,7 +6,7 @@ metadata:
     cert-manager.io/cluster-issuer: letsencrypt-prod
     dns.gardener.cloud/dnsnames: 'auth.hub.tekton.dev'
     dns.gardener.cloud/ttl: "3600"
-  name: auth
+  name: tekton-hub-auth
```
Comment (vinamra28) [inline config/04-kubernetes/40-auth-ingress.yaml:9]: can you please modify the line https://github.com/tektoncd/hub/blob/0937bdd758edcafc8599859c60074a6e9660cfa2/config/04-kubernetes/40-auth-ingress.yaml#L16 as well?

File: config/04-kubernetes/42-ui-ingress.yaml [hunk 2]
```diff
@@ -1,7 +1,7 @@
 apiVersion: extensions/v1beta1
 kind: Ingress
 metadata:
-  name: ui
+  name: tekton-hub-ui
```
Comment (vinamra28) [inline config/04-kubernetes/42-ui-ingress.yaml:4]: same as above

File: config/04-kubernetes/40-api-ingress.yaml [hunk 3]
```diff
@@ -6,7 +6,7 @@ metadata:
     cert-manager.io/cluster-issuer: letsencrypt-prod
     dns.gardener.cloud/dnsnames: 'api.hub.tekton.dev'
     dns.gardener.cloud/ttl: "3600"
-  name: api
+  name: tekton-hub-api
```
Comment (vinamra28) [inline config/04-kubernetes/40-api-ingress.yaml:9]: same as above

**Review by vinamra28 — CHANGES_REQUESTED** [review]

**Comment by sm43** [comment]
this changes won't be backward compatible right? 🤔

**Comment by vinamra28** [comment]
> this changes won't be backward compatible right? thinking

nope

File: config/00-init/02-db-pvc.yaml [hunk 4]
```diff
@@ -15,9 +15,9 @@
 apiVersion: v1
 kind: PersistentVolumeClaim
 metadata:
-  name: db
+  name: tekton-hub-db
   labels:
-    app: db
+    app: tekton-hub-db
```
Comment (vinamra28) [inline config/00-init/02-db-pvc.yaml:20]: do we want to change the value of the labels as well?
cc @PuneetPunamiya @piyush-garg @sm43

**Comment by sm43** [comment]
we need to document this, how to replace existing deployment with this changes
and include in release notes

**Pull request merged**